### DISCUSSION TITLE
Cannot join any jobs

### DISCUSSION THREAD

**Discussion by aoqia194** [YimMenu/YimMenu#858] [Q&A] [answered]
When I join a job (I am not host), and go to start the job, I get kicked straight out of the job screen and back into freemode. No clue why.

Disabled all protections and it didn't help. Restarted game. Made sure to use main YimMenu/YimMenu github.
When loading back into freemode, game has overlayed distance blur effect post effect too.

For example to replicate, try going into quick job and search for one on call, then go in and wait til they ready the job. You will get kicked.

**Comment by aoqia194** [comment, marked as answer]
It was Force Script Host that kept kicking me.

**Reply by maybegreat48** [reply]
Might be "Kick Host During Join"

**Reply by aoqia194** [reply]
> Might be "Kick Host During Join"

It turns out to be "Force Script Host". I didn't think to disable that and had kept it on the whole time.
